Allow super users to update shared flag on zones (#410)
vinyldns/vinyldns: modules/docs/src/main/tut/api/zone-model.md

email         | string      | The distribution email for the zone |
connection    | ZoneConnection | The connection used to issue DDNS updates to the backend zone.  If not provided, default keys will be used.  See the [Zone Connection Attributes](#zone-conn-attr) for more information |
transferConnection | ZoneConnection | The connection that is used to sync the zone with the DNS backend.  This can be different than the update connection.  If not provided, default keys will be used |

> shared        | boolean     | An indicator that the zone is shared with anyone. |

Reviewer: Should we indicate this has limited use at this time? Like it can only be set by super users.

Author: That's a good suggestion - I'll add a tidbit about who can set/modify this.

> shared        | boolean     | An indicator that the zone is shared with anyone. Can only be set and/or modified by super users. Default value is `false`. |

Reviewer 2: actually, can you split these docs changes into a separate PR? i wanna wait on that merge until our support is complete

acl           | ZoneACL     | The access control rules governing the zone.  See the [Zone ACL Rule Attributes](#zone-acl-rule-attr) for more information
id            | string      | The unique identifier for this zone

> isTest        | boolean     | An indicator that the zone is a zone for testing purposes. |

Reviewer: ```suggestion
 isTest        | boolean     | Indicates the zone is intended for testing purposes only if true. |
```

email         | string      | The distribution email for the zone |
connection    | ZoneConnection | The connection used to issue DDNS updates to the backend zone.  If not provided, default keys will be used.  See the [Zone Connection Attributes](#zone-conn-attr) for more information |
transferConnection | ZoneConnection | The connection that is used to sync the zone with the DNS backend.  This can be different than the update connection.  If not provided, default keys will be used |

> <<<<<<< HEAD

Reviewer: remove

shared        | boolean     | An indicator that the zone is shared with anyone. Can only be set and/or modified by super users. Default value is `false`. |
acl           | ZoneACL     | The access control rules governing the zone.  See the [Zone ACL Rule Attributes](#zone-acl-rule-attr) for more information
id            | string      | The unique identifier for this zone
isTest        | boolean     | An indicator for whether the zone is intended for testing purposes. |

> =======

Reviewer: remove

shared        | boolean     | **DEPRECATED** An indicator that the zone is shared with anyone.  No longer used as ACLs are now used to manage zones. |
acl           | ZoneACL     | The access control rules governing the zone.  See the [Zone ACL Rule Attributes](#zone-acl-rule-attr) for more information
id            | string      | The unique identifier for this zone

> >>>>>>> parent of e6201006... Update docs.

Reviewer: remove
